packages/table/src/table.tsx

```diff
@@ -51,6 +53,14 @@ export interface ITableProps extends IProps, IRowHeights, IColumnWidths {
     fillBodyWithGhostCells?: boolean;
 
     /**
+     * If exists, a callback that returns the data for a specific cell. This need not
+     * match the value displayed in the `<Cell>` component. The value will be
+     * invisibly added as `textContent` into the DOM before copying. If not exists,
+     * copy via hotkeys (mod+c) will not work.
```

Comment:
But we could consider renaming this `getCellClipboardData` since without the naming, users might think this is used to fill cells when rendering.